localstack-core/localstack/services/lambda_/invocation/version_manager.py

```diff
@@ -212,6 +213,16 @@ def invoke(self, *, invocation: Invocation) -> InvocationResult:
                 self.store_logs(
                     invocation_result=invocation_result, execution_env=ldm_execution_environment
                 )
+            except CancelledError:
+                # Timeouts for invocation futures are managed by LDM, a cancelled error here is
```

Comment:
We set a timeout, right? it's just a very long one, but it still might have actually run out. Maybe we can include this in the error message?